packages/suite/src/components/suite/SecurityCheck/DeviceCompromised.tsx

```diff
@@ -45,12 +26,6 @@ export const DeviceCompromised = () => {
         }
     };
 
-    useEffect(() => {
-        const commonData = { revision, version, vendor };
-        if (revisionCheckError) reportCheckFail('revision', { ...commonData, revisionCheckError });
-        if (hashCheckError) reportCheckFail('hash', { ...commonData, hashCheckError });
-    }, [revisionCheckError, hashCheckError, revision, vendor, version]);
```

Comment:
This had a problem: when only one of the error changes, it re-reports both of them